src/mvba/bundle.rs

```diff
@@ -11,7 +11,14 @@ pub struct Bundle {
     /// This is the destination  module, it can be ABBA, VCBC or MVBA.
     pub module: String,
```

Comment:
The module string is no longer needed. We can match on the message enum to decide which module to dispatch the message to.